### ISSUE TITLE
ShowPeopleDesc not working

### ISSUE THREAD

**Issue by esackbauer** [immichFrame/ImmichFrame#204]
### Version

1.0.18.0

### Deployment Method

Web: Docker compose, Windows Client

### Description

Whatever I put in docker compose file as env variable for "ShowPeopleDesc", if I set it to false, no people description should be shown.
However it is still shown, even after changing the env variables and docker compose down, docker compose up -d
And restarting the Client App.
### 

### Reproduction

Parameter "ShowPeopleDesc" is missing in docker compose template.
After adding it, and setting it to false, redeploy docker container with docker compose up -d
Restart Immichframe Windows client
People's names are still shown.

### Pre-Submission Checklist

- [X] This is a bug report and not a feature request

- [X] I have provided all of the required information to reproduce the bug (config, logs, etc.)

- [X] I have checked for related issues and checked the documentation

**Comment by JW-CH** [comment]
Hey @esackbauer, you are probably using a `album`-filter, right? We do not get information about the people from the immich api when doing filtering on our side. Sadly, this is a limitation of the immich api (as of now). Can you remove the `people`, `album`, and `showMemories` filters (if set up) and check if `ShowPeopleDesc` is working? Just to confirm this is not an issue on our side.

**Comment by esackbauer** [comment]
> Hey @esackbauer, you are probably using a `album`-filter, right?

No, there is no filter active, see here my env-config:
```
    environment:
      TZ: "Europe/Berlin"
      # Required
      ImmichServerUrl: "http://10.0.0.11:2283"
      ApiKey: "MmbNQuZiQRvaReLY0C4QDanEZ3THl7gchs6Fo2eNE"
      # Image
      ImageZoom: "false"  
      Interval: "10"
      TransitionDuration: "2"
      # Filters
      # Albums: "ALBUM1,ALBUM2"
      #ExcludedAlbums: "Archive"
      # People: "PERSON1,PERSON2"
      ShowMemories: "false"
      ShowPeopleDesc: "false"
      #ImagesFromDays: ""
      #ImagesFromDate: ""
      #ImagesUntilDate: ""
      # Clock
      ShowClock: "false"
      ClockFormat: "HH:mm"
      # Weather
      #WeatherApiKey: "API-KEY"
      #UnitSystem: "imperial"
      #Language: "en"
      #ShowWeatherDescription: "false"
      #WeatherLatLong: "40.730610, -73.935242"
      # Metadata
      ShowImageDesc: "false"
      ShowImageLocation: "true"
      ShowPhotoDate: "true"
      PhotoDateFormat: "yyyy-MM-dd"
      # Caching
      RenewImagesDuration: "30"
      DownloadImages: "false"
      RefreshAlbumPeopleInterval: "12"
      # UI
      PrimaryColor: "#FF5733"
      # Misc
      #ImmichFrameAlbumName: ""
```

**Comment by esackbauer** [comment]
> Hey @esackbauer, you are probably using a `album`-filter, right? 

No, there is no filter active, see here my env-config:
´´´
    environment:
      TZ: "Europe/Berlin"
      # Required
      ImmichServerUrl: "http://asdasd:2283"
      ApiKey: "asdasd"
      # Image
      ImageZoom: "false"  
      Interval: "10"
      TransitionDuration: "2"
      # Filters
      # Albums: "ALBUM1,ALBUM2"
      #ExcludedAlbums: "Archive"
      # People: "PERSON1,PERSON2"
      ShowMemories: "false"
      ShowPeopleDesc: "false"
      #ImagesFromDays: ""
      #ImagesFromDate: ""
      #ImagesUntilDate: ""
      # Clock
      ShowClock: "false"
      ClockFormat: "HH:mm"
      # Weather
      #WeatherApiKey: "API-KEY"
      #UnitSystem: "imperial"
      #Language: "en"
      #ShowWeatherDescription: "false"
      #WeatherLatLong: "40.730610, -73.935242"
      # Metadata
      ShowImageDesc: "false"
      ShowImageLocation: "true"
      ShowPhotoDate: "true"
      PhotoDateFormat: "yyyy-MM-dd"
      # Caching
      RenewImagesDuration: "30"
      DownloadImages: "false"
      RefreshAlbumPeopleInterval: "12"
      # UI
      PrimaryColor: "#FF5733"
      # Misc
      #ImmichFrameAlbumName: ""
```

**Comment by JW-CH** [comment]
`ShowPeopleDesc: "false"`
You have this in your settings. Can you turn that on or leave it empty?

**Comment by esackbauer** [comment]
I have tried that, and the people's names are still shown.

**Comment by JW-CH** [comment]
I am sorry, the item #197 was not available in v1.0.18.0. I created a new release including this setting. You should be able to update your compose, and it should work with your settings. Sorry. https://github.com/immichFrame/ImmichFrame/releases/tag/v1.0.19.0

**Comment by 3rob3** [comment]
I was wondering about that 😄
